examples/knowledge/exampledata/ast/proto-lib/agent/v1/agent.proto

```
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package agent.v1;
```

Buf (1.67.0)

[error] Files with package "agent.v1" must be within a directory "agent/v1" relative to root but were in directory "examples/knowledge/exampledata/ast/proto-lib/agent/v1".

(PACKAGE_DIRECTORY_MATCH)